src/common/payment.ts

```diff
@@ -190,7 +194,7 @@ export class TurboAuthenticatedPaymentService
     return this.httpService.get<TurboWincForFiatResponse>({
       endpoint: `/price/${amount.type}/${
```

Comment:
URL builder classes just eliminate this whole class of errors. We really should be using them around more of our codebases. I think this is the second time I've mentioned that.

Reply:
+1 - we made same hiccups on arns.app before moving to URL builder

Reply:
agreed and noted. the promo code helper fn I added for this isn't adding much usefulness